### PR TITLE
Tt us17 update

### PR DESCRIPTION
**Pull request by tylertomlinson**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [x] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [x] What stories does the PR include

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** = 100% 
--------------------------------------------------------------------------------
### Change Log
Resolves issue US17 #22 

Updated item index to have active_items instead of all items 
Updated tests

### CODE REVIEW

**Review by Mariana-21 — APPROVED** [review]

File: app/models/item.rb [hunk 1]
```diff
@@ -12,6 +12,10 @@ class Item <ApplicationRecord
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
 
+  def self.active_items
+    where(active?: true)
+  end
```
Comment (alex-latham) [inline app/models/item.rb:17]: We'll need a model test or else coverage in spec/models drops to 99.74%

File: app/models/item.rb [hunk 2]
```diff
@@ -12,6 +12,10 @@ class Item <ApplicationRecord
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
 
+  def self.active_items
+    where(active?: true)
```
Comment (danielpselph) [inline app/models/item.rb:16]: Effective use of helper method

**Review by danielpselph — APPROVED** [review]
Changes approved

**Review by alex-latham — APPROVED** [review]
Looking good to me

**Pull request merged**